src/components/inputs/ChipInput.test.tsx

```diff
@@ -844,4 +844,67 @@ describe('ChipInput', () => {
 		}
 		expect(screen.getAllByText(/chip/)).toHaveLength(prevLimitMaxPlusOne);
 	});
+
+	it('should give TRUE value on onOptionsDisplayChange when the user opens the dropdown by clicking the input element', async () => {
+		const onOptionsDisplayChangeFn = jest.fn();
+		const initial = [
```

Comment:
Give a type to this object, it is using fields that does not belong to the ChipInput